doc/releases/changelog-dev.md

```diff
@@ -288,7 +451,7 @@
 
 This release contains contributions from (in alphabetical order):
 
-Venkatakrishnan AnushKrishna
+Venkatakrishnan AnushKrishna,
```

Comment:
Could you please add `Modjtaba Shokrian-Zini` for adding non-cubic resource estimation [PR](https://github.com/PennyLaneAI/pennylane/pull/3956)?